docs/insights-archive-sample/insights-operator/conditional-gatherer-rules.json

```diff
@@ -192,5 +192,11 @@
       "errors": null,
       "was_triggered": false
     }
+  ],
+  "endpoint": "https://console.redhat.com/api/gathering/gathering_rules",
+  "firing_alerts": [
+    "AlertmanagerClusterDown",
+    "TargetDown",
+    "Watchdog"
```

Comment:
Isn't that redundant information,please? I mean we have `was_triggered` flag, which implies that an alert was firing right?

Reply:
It can be helpful for debugging and also could be used in the integration tests later

Reply:
Having something helpful for debugging and in the integration tests doesn't mean we have to add it to the archive ....at least for me :-P Anyway, it's probably not a big deal (although once something is in the archive then it's pretty difficult to remove it), I am just saying that I think the information is already there.

Reply:
It's not too important tho, we could delete it if you want

Reply:
Good point with the conditions potentially being more complicated, but we don't have any at the moment AFAIK so I wouldn't add it now.

Reply:
changed it

Reply:
Thanks so we need to change the sample archive as well right.

Reply:
oops